### ISSUE TITLE
Clarify in 3.3.8 Redundant Entry understanding document "available for the user to select."

### ISSUE THREAD

**Issue by mraccess77** [w3c/wcag#1973]
The phrase "available for the user to select." can be taken multiple ways and I assume all are acceptable - but it would be good to make this clear in the understanding document.

* Available for the user to select and option to populate the redundant data
* Available for the user to select, copy, and paste the redundant data
* Available for the user to select a record that has the data already provided (example from technique).

**Comment by alastc** [comment]
Hi Jon,

What's the difference between the 1st and 3rd instances? They look very similar to me, although perhaps I'm missing a scenario.

-Alastair

**Comment by mraccess77** [comment]
The different might be 3 is a complete record you are selecting and 1 is an aspect such as a field or fields to populate.    The technique gives some good examples about checking a box to bring over shipping details into billing details for example.    Basically filling a field or section on demand rather than bring over a whole pre-completed record.

**Comment by alastc** [comment]
#1989 is open for suggestions, wasn't accepted yet.
